lib/components/find-a-domain-component.js

```diff
@@ -10,64 +10,80 @@ export default class FindADomainComponent extends BaseContainer {
 		super( driver, By.css( '.register-domain-step' ) );
 		this.declineGoogleAppsLinkSelector = By.className( 'google-apps-dialog__cancel-link' );
 	}
+
 	waitForResults() {
 		const driver = this.driver;
 		const resultsLoadingSelector = By.css( '.domain-suggestion.is-placeholder' );
 		driver.wait( function() {
 			return driverHelper.isElementPresent( driver, resultsLoadingSelector ).then( function( present ) {
-				return ! present;
+				return !present;
 			} );
 		}, this.explicitWaitMS * 2, 'The domain results loading element was still present when it should have disappeared by now.' );
 		return this.checkForUnknownABTestKeys();
 	}
+
 	waitForGoogleApps() {
 		return this.driver.wait( until.elementLocated( this.declineGoogleAppsLinkSelector ), this.explicitWaitMS, 'Could not locate the link to decline google apps.' );
 	}
+
 	searchForBlogNameAndWaitForResults( blogName ) {
 		const searchInputSelector = By.className( 'search__input' );
 		driverHelper.setWhenSettable( this.driver, searchInputSelector, blogName );
 		return this.waitForResults();
 	}
+
 	checkAndRetryForFreeBlogAddresses( expectedBlogAddresses, blogName ) {
 		const self = this;
 		self.freeBlogAddress().then( ( actualAddress ) => {
-			if ( expectedBlogAddresses.indexOf( actualAddress ) < 0 ) {
+			if (expectedBlogAddresses.indexOf( actualAddress ) < 0) {
 				let message = `The displayed free blog address: '${actualAddress}' was not in the expected addresses: '${expectedBlogAddresses}'. Re-searching for '${blogName}' now.`;
 				slackNotifier.warn( message );
 				self.searchForBlogNameAndWaitForResults( blogName );
 			}
 		} );
 	}
+
 	freeBlogAddress() {
 		const freeBlogAddressSelector = By.css( '.domain-suggestion__content h3' );
 		return this.driver.findElement( freeBlogAddressSelector ).getText().then( function( text ) {
 			return text;
 		} );
 	}
+
 	selectDotComAddress( dotComAddress ) {
 		const selector = By.css( `[data-e2e-domain="${ dotComAddress }"]` );
 		this.driver.wait( until.elementLocated( selector ), this.explicitWaitMS, 'Could not locate the select button for the paid address: ' + dotComAddress );
 		const element = this.driver.findElement( selector );
 		this.driver.wait( until.elementIsEnabled( element ), this.explicitWaitMS, 'The paid address button for ' + dotComAddress + ' does not appear to be enabled to click' );
 		return driverHelper.clickWhenClickable( this.driver, selector, this.explicitWaitMS );
 	}
+
 	selectFreeAddress() {
 		const freeAddressSelector = By.css( '.domain-suggestion.is-clickable' );
 		return driverHelper.clickWhenClickable( this.driver, freeAddressSelector, this.explicitWaitMS );
 	}
+
 	selectMapOwnDomain() {
```

Comment:
It looks like `selectMapOwnDomain()` is unused, so you could possibly update and use this, or remove it?

Reply:
I removed `selectMapOwnDomain()` here because I think it makes more sense. If I am not mistaken, prior to the introduction of Incoming Domain Transfers, the Domain Mapping section was on the same page where you would enter a new domain name if you were looking to register one. Now, there is no mention of the Domain Mapping immediately on the page where you would register a new domain name so I think it is best to keep all domain mapping related functions in a separate component dedicated to domain mapping specifically.